src/assets/i18n/en_US.json

```diff
@@ -70,6 +72,66 @@
   "account.company_profile.name.label": "Company Name",
   "account.company_profile.taxation_id.label": "Taxation ID",
   "account.companyprofile.taxationid.label": "Taxation ID",
+  "account.costCenter.active.label": "Active",
+  "account.costCenter.status.inactive": "(Inactive)",
+  "account.costcenter.activate.button.title": "Activate",
+  "account.costcenter.budget.error.required": "Please enter a money value (Budget).",
+  "account.costcenter.budget.error.valid": "Please enter a valid money value (Budget).",
+  "account.costcenter.budget.label": "Budget",
+  "account.costcenter.budget.period.value.fixed": "fixed",
+  "account.costcenter.budget.period.value.monthly": "monthly",
+  "account.costcenter.budget.period.value.quarterly": "quarterly",
+  "account.costcenter.budget.period.value.weekly": "weekly",
+  "account.costcenter.create.button.label": "Create Cost Center",
+  "account.costcenter.create.heading": "Add Cost Center",
+  "account.costcenter.deactivate.button.label": "Deactivate",
+  "account.costcenter.deactivate.button.title": "Deactivate",
+  "account.costcenter.deactivate.question": "Do you really want to deactivate this cost center?",
+  "account.costcenter.deactivate.text": "There are {{1}} orders and {{0}} pending orders in cost center {{2}}. - Deactivating the cost centers has no effect on pending orders or orders that have already been placed.",
```

Comment:
Would it be possible to display singular of the word "order" in case it is only one? Else I would place the plural "s" in brackets so that it fits grammatically in any case.